src/main/java/com/labsynch/labseer/service/BulkLoadServiceImpl.java

```diff
@@ -1793,9 +1783,15 @@ public Lot addLotAlias(Lot lot, String lsType, String lsKind,
 			String lookUpProperty, String lookUpString) {
 
 		if (lookUpString != null) {
-			// found LiveDesign Corp Name
-			logger.info("Found one or more lot alias: " + lookUpProperty + "  " + lookUpString);
-			String[] aliases = lookUpString.split(";");
+			// Split on semicolon and trim
```

Comment:
nit* - Because we are doing this twice, can we wrap it in a helper function?

Reply:
Yep, done!